FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Tool/appsettings.json

```diff
@@ -23,7 +23,7 @@
     "enableExternalFilter": false,
     "filterImageReference": "",
     "filterConfigurationFileName" : "",
-    "filterScope": "System",
+    "filterScope": "sys",
```

Comment:
In the future, we'll deprecate `System` and use `sys` instead?

Reply:
Sorry, it's a mis-uploading.